### PR TITLE
[INTGW-792] unused methods removed

### PR DESCRIPTION
**Pull request by chathurabuddi**

### CODE REVIEW

File: components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbUtils.java [hunk 1]
```diff
@@ -389,7 +389,7 @@ public static String format(Double doubleData, int precision, int scale) throws
      *
      * @return the connection
      * @throws Exception the exception
-     */
+     *
     public Connection connect() throws Exception {
         System.out.println("-------- JDBC Connection Init ------------");
         Connection connection = null;
```
Comment (cjpannila) [inline components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbUtils.java]: If the methods are not being used, shouldn't those be removed from the code without keeping commented lines.

File: components/dbutil/src/main/java/com/wso2telco/dbutils/DbUtils.java [hunk 2]
```diff
@@ -330,7 +330,7 @@ public void disconnect(Connection con) throws Exception {
         System.out.println();
         // System.out.println(" Disconnect from database.");
 
-        // makes all changes made since the previous commit/rollback permanent
+        // makes all cconnecthanges made since the previous commit/rollback permanent
```
Comment (cjpannila) [inline components/dbutil/src/main/java/com/wso2telco/dbutils/DbUtils.java]: What is the purpose of this change.

File: components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbUtils.java (hunk 1 above)
Comment (chathurabuddi) [reply components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbUtils.java]: removed.

File: components/dbutil/src/main/java/com/wso2telco/dbutils/DbUtils.java (hunk 2 above)
Comment (chathurabuddi) [reply components/dbutil/src/main/java/com/wso2telco/dbutils/DbUtils.java]: It was a mistake, fixed
